slo_generator/api/main.py

```diff
@@ -191,7 +191,7 @@ def process_batch_req(request, data, config):
     for url in urls:
         if 'pubsub_batch_handler' in config:
             LOGGER.info(f'Sending {url} to pubsub batch handler.')
-            from google.cloud import pubsub_v1 # pylint: disable=C0415
+            from google.cloud import pubsub_v1
```

Comment:
For the reasons discussed above, I'd rather keep this annotation. It indicates something might be wrong and we could address it properly in a different PR.